libs/core/algorithms/include/hpx/parallel/algorithms/reverse.hpp

```diff
@@ -146,7 +146,7 @@ namespace hpx {
     ///                     the algorithm will be applied to.
     /// \param last         Refers to the end of the sequence of elements the
     ///                     algorithm will be applied to.
-    /// \param dest         Refers to the begin of the destination range.
+    /// \param dest_first   Refers to the begin of the destination range.
```

Comment:
I think the parameter name used here needs to correspond to the name used in the function signature below. 
```suggestion
    /// \param first        Refers to the begin of the destination range.
```